File: c/6.c
#include <stdio.h>

int main(void)
{
	int i, sumsq = 0, sqsum = 50 * 50 * 101 * 101;

	for (i = 1; i <= 100; ++i)
		sumsq += i * i;

	printf("%d\n", sqsum - sumsq);
	return 0;
}

File: clj/6.clj
(defn sumsq [n]
  (apply + (map #(* % %) (range 1 (+ 1 n)))))

(defn sqsum [n]
  (#(* % %) (apply + (range 1 (+ 1 n)))))

(prn (- (sqsum 100) (sumsq 100)))
